scripted/sbt/src/main/scala/sbt/test/SbtHandler.scala

```diff
@@ -80,7 +80,10 @@ final class SbtHandler(directory: File,
     val thread = new Thread() { override def run() = { p.exitValue(); server.close() } }
```

Comment:
We can potentially discuss it, but I'm relatively happy with Travis CI as we get 5 workers for free. I think that's pretty good ROI.